### ISSUE TITLE
Support autowiring for Symfony 4

### ISSUE THREAD

**Issue by gander** [getsentry/sentry-symfony#157]
I would like to use autowiring in Symfony 4 without having to enter it in the service configuration:

```yaml
services:
    Sentry\SentryBundle\SentrySymfonyClient: '@sentry.client'
```

```php
<?php

namespace App\Controller;

use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
use Sentry\SentryBundle\SentrySymfonyClient;
use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\Routing\Annotation\Route;

class DefaultController extends AbstractController
{
    /**
     * @Route("/", name="homepage")
     * @param SentrySymfonyClient $client
     * @Template()
     */
    public function index(SentrySymfonyClient $client)
    {
    }
}
```

**Comment by Jean85** [comment]
Good idea, PR welcome on this!

**Comment by Jean85** [comment]
This is solved by #158.
